buildscripts/observability-test/Dockerfile

```diff
@@ -17,11 +17,18 @@
 # Stage 1: Build the interop test client and server
 #
 
-FROM openjdk:11.0.16-jdk-slim-bullseye AS build
+FROM eclipse-temurin:11-jdk AS build
 
 WORKDIR /grpc-java
 COPY . .
 
+# Intentionally after the app COPY to force the update on each build.
```

Comment:
This doesn't do what you think it does, because this isn't part of the final image.